b2b-app/imports/ui/forms/survey-builder/recoil/atoms/parts-state.js

```diff
@@ -25,3 +31,182 @@ export const partsAtom = atom({
     },
   ],
 })
+
+const defaultPart = { name: '' }
+
+export const partAtom = atomFamily({
```

Comment:
part family: when editing a question, it will only change the state for that part, not all the questions.